chrome-extension/test/configure.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+process.exit();
```

Comment:
Could you change the commit message in https://github.com/haystack/tipsy/pull/91/commits/4989d57b71641725e506f1095b183e7effb8307c to "Make chrome test less broken" (or something like that). It seems like a strict improvement but the commit message is a bit misleading.